de-lib/src/main/java/org/iplantc/de/apps/client/presenter/AppsViewPresenterImpl.java

```diff
@@ -94,13 +95,15 @@ public AppCategory getSelectedAppCategory() {
 
     @Override
     public void go(final HasOneWidget container,
-                   final HasId selectedAppCategory,
-                   final HasId selectedApp) {
+                   final HasId selectedAppCategory, final HasId selectedApp, final String actview) {
```

Comment:
These parameters should probably all be on separate lines for consistency. Also, we should probably rename the variable to `activeView` with camelCase.

I think there are also some tests for this class so you might want to check for breakages.